src/oc/web/components/ui/login_overlay.cljs

```diff
@@ -305,20 +305,20 @@
   [s]
   (cond
     ; login via email
-    (or (= (drv/react s user-store/show-login-overlay-key) :login-with-email)
-        (= (drv/react s user-store/show-login-overlay-key) :login-with-slack))
+    (or (= (drv/react s dis/show-login-overlay-key) :login-with-email)
+        (= (drv/react s dis/show-login-overlay-key) :login-with-slack))
     (login-with-email)
     ; signup via email
-    (or (= (drv/react s user-store/show-login-overlay-key) :signup-with-email)
-        (= (drv/react s user-store/show-login-overlay-key) :signup-with-slack))
+    (or (= (drv/react s dis/show-login-overlay-key) :signup-with-email)
+        (= (drv/react s dis/show-login-overlay-key) :signup-with-slack))
     (do
       (utils/after 150 #(router/nav! oc-urls/sign-up))
       [:div])
     ; password reset
-    (= (drv/react s user-store/show-login-overlay-key) :password-reset)
+    (= (drv/react s dis/show-login-overlay-key) :password-reset)
     (password-reset)
     ; form to insert a new password
-    (= (drv/react s user-store/show-login-overlay-key) :collect-password)
```

Comment:
We should be consistent here.  Do we want to have the stores determine the method of retrieving information from the app state or use one large dispatch namespace to do that. I think having them in the store is the best.

Reply:
Well if we want to be consistent we should keep the keys in dispatcher, they are all there, that's why i moved this too.

We are keeping all the keys in dispatcher to avoid pulling stores to all the files the need that key or another. You only need to pull one file (dispatcher) to have the keys available.
Then the setter are in the stores.

I know that ideally is better to have the keys in the stores to avoid having a huge namespace but since some keys are using in more stores it's not very easy to change it now.
This is an example of it: the `show-login-overlay-key` is needed not only by the user store but also from the jwt store (that i created in this PR to pull out the JWT stuff to make it callable from the ws clients namespaces), if we keep it in the user store i need to pull it in the jwt store that is creating a circular dependency cycle.

I know you changed this but this is the only file that is keeping the keys inside, so: or we refactor all or we keep it consistent like it is right now.

I think at this point is cleaner this way.